keyboards/soy20/keymaps/default/keymap.c

```diff
@@ -0,0 +1,33 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
+		KC_PMNS, KC_PAST, KC_PSLS, MO(1),
+		KC_PPLS, KC_P9, KC_P8, KC_P7,
+		KC_TAB, KC_P5, KC_P6, KC_P4,
+		KC_PENT, KC_P3, KC_P2, KC_P1,
+		KC_DEL, KC_SCLN, KC_PDOT, KC_P0),
+
+	[1] = LAYOUT(
+		KC_NLCK, KC_PSCR, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, RESET),
+
+	[2] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	[3] = LAYOUT(
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
```

Comment:
```suggestion
```

Reply:
Empty Layouts 3 and 4 are required for VIA support.

Reply:
Yes, in the `via` keymap. This is the default.

Reply:
Ahh Thank you, my mistake.